src/views/Notifier/components/NotifyForm/index.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useEffect, useState } from 'react';
+import {
+  ActionGroup,
+  Button,
+  ButtonVariant,
+  Form,
+} from '@patternfly/react-core';
+import FormHeading from 'components/FormHeading';
+import SearchForm from './components/SearchForm';
+import SearchResultsList from './components/SearchResultsList';
+
+/** @typedef {import('api').Parcel} Parcel */
+
+/**
+ * Form for editing a parcel's risk
```

Comment:
This form is not used for editing a parcel's risk. Its purpose is to search for surrounding parcels within a specified radius to be notified.